packages/controller/controller.ts

```diff
@@ -317,20 +316,19 @@ async function initialize(): Promise<InitializeParameters> {
 	logger.info("Loading Plugin info");
 	const pluginInfos = await lib.loadPluginInfos(pluginList);
 	lib.registerPluginMessages(pluginInfos);
-	lib.registerPluginConfigGroups(pluginInfos);
-	lib.finalizeConfigs();
+	lib.addPluginConfigFields(pluginInfos);
```

Comment:
It more closely describes what it's doing. Previously it would call `registerGroup` so register made sense. Now it adds the fields to the configs so add makes sense.